### PR TITLE
Feat/new pdf layout

### PR DESCRIPTION
**Pull request by jona159**
This PR introduces the new PDF Layout as presented in [asana](https://app.asana.com/0/1205782519509072/1206765520450288/1206951291894533)

### CODE REVIEW

**Comment by jona159** [comment]
Dont merge yet

File: package.json
Comment (timber-they) [inline package.json:57]: Why do we need an older version?

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:6]: I think we typically do use the curly braces, so we should probably stick to it to stay consistent

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:13]: Also curly braces here

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:209]: Could this cause problems if the first or last name has a length of 0?

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:201]: This variable name could be more descriptive / precise, e.g. `backgroundImage`

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:250]: There's a lot of "magic numbers" here, but I assume it would be hard to extract them into meaningful constants or something like that?

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:268]: Would it make sense to have the text not hardcoded here? I imagine this could be an easy spot to miss if the text of the PDF was ever to be changed

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:238]: Maybe the "add page" part (so the whole body of this `if`) could be extracted? This method is already very long

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:303]: Maybe instead of just commenting the different sections in this method, each section could have its own method? So something like `addOebLogo`, `addName` etc.. I believe this could make this huge method more readable

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:382]: Commented code

**Review by timber-they — CHANGES_REQUESTED** [review]

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:6]: using curly braces again

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts]: made an if check for the length of 0

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts]: changed the variable name

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts]: not sure if i would be able to find a good name for everything

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:300]: Maybe we can redo this when we add translations?

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts]: deleted

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:13]: Huh that's weird, but ok

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:300]: Ok, but then maybe a TODO comment would be good?

Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:411]: you mean in this specific place or everywhere where text is hardcoded? But maybe it is easier to spot now, that every section is in a function

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts]: Extracted it

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (timber-they) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:411]: I meant in this specific place, so that we don't forget it

**Comment by timber-they** [comment]
Also, currently the build seems to fail

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts
Comment (jona159) [reply src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:412]: added it

**Comment by timber-they** [comment]
@jona159 after resolving the merge conflicts this can be merged

**Review by timber-they — APPROVED** [review]

**Comment by timber-they** [comment]
From my point of view you can merge this, but there's still the "Don't merge yet" comment

**Comment by jona159** [comment]
> From my point of view you can merge this, but there's still the "Don't merge yet" comment

@timber-they After merging main into this branch the pdf preview in chrome was broken (still worked in firefox) due to the larger image dimensions of the new ```Logo-New.png```. As a workaround i resized this image and added a new file ```Logo-New-Small.png```, but now we have a duplicate logo file. Going to merge for now, but do you think it is possible only to use the image with the smaller dimensions and remove the ```Logo-New.png```, so that we dont have a duplicate?

**Pull request merged**
